### DISCUSSION TITLE
carbon upgrade search for typescript files

### DISCUSSION THREAD

**Discussion by pmweeks98** [carbon-design-system/carbon#14865] [Ideas]
Currently the upgrade package only seaches for js and jsx files - it should be extended to include ts and tsx files

**Comment by tay1orjones** [comment]
I agree! None of the existing fixtures and test cases cover ts or tsx files. I'm not sure how the existing codemods would run against these filetypes. There may be errors or edge cases not caught. If you have a chance to test, we'd welcome any improvements to the upgrade package like this.
